<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-IEOF3HT" ignored="8" name="Default test" tests="1" failures="1" timestamp="05 Oct 2017 12:03:51 GMT" time="12.683" errors="0">
  <testcase name="validateZipCodeLookUp" time="0.22" classname="com.infrrd.roostify.testcase.HomePageTestCase">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.infrrd.roostify.testcase.HomePageTestCase.validateZipCodeLookUp(HomePageTestCase.java:102)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- validateZipCodeLookUp -->
  <testcase name="validateOtherOptimizations" time="0.0" classname="com.infrrd.roostify.testcase.HomePageTestCase">
    <ignored/>
  </testcase> <!-- validateOtherOptimizations -->
  <testcase name="validateGraphLegends" time="0.0" classname="com.infrrd.roostify.testcase.HomePageTestCase">
    <ignored/>
  </testcase> <!-- validateGraphLegends -->
  <testcase name="checkEnablingOfGetMyOptionsButton" time="0.0" classname="com.infrrd.roostify.testcase.HomePageTestCase">
    <ignored/>
  </testcase> <!-- checkEnablingOfGetMyOptionsButton -->
  <testcase name="validateZipCodeError" time="0.0" classname="com.infrrd.roostify.testcase.HomePageTestCase">
    <ignored/>
  </testcase> <!-- validateZipCodeError -->
  <testcase name="validateOptionsForPurchase" time="0.0" classname="com.infrrd.roostify.testcase.HomePageTestCase">
    <ignored/>
  </testcase> <!-- validateOptionsForPurchase -->
  <testcase name="validateOptionsForRefinance" time="0.0" classname="com.infrrd.roostify.testcase.HomePageTestCase">
    <ignored/>
  </testcase> <!-- validateOptionsForRefinance -->
  <testcase name="validatingLeadForm" time="0.0" classname="com.infrrd.roostify.testcase.HomePageTestCase">
    <ignored/>
  </testcase> <!-- validatingLeadForm -->
  <testcase name="validateOptionsForRefinanceCashout" time="0.0" classname="com.infrrd.roostify.testcase.HomePageTestCase">
    <ignored/>
  </testcase> <!-- validateOptionsForRefinanceCashout -->
</testsuite> <!-- Default test -->
